plugins/npm/nexus-npm-repository-plugin/src/main/java/com/bolyuba/nexus/plugin/npm/hosted/RecreateMetadataWalkerProcessor.java

```diff
@@ -191,9 +193,21 @@ public void onCollectionExit(final WalkerContext context, final StorageCollectio
       // do this in memory, as package.json is expected to be fairly small (few kb at most)
       final ByteArrayOutputStream bos = new ByteArrayOutputStream(Ints.checkedCast(tarEntry.getSize()));
       StreamSupport.copy(tarArchiveInputStream, bos, StreamSupport.BUFFER_SIZE);
-      final Map<String, Object> packageJson = objectMapper
-          .readValue(bos.toByteArray(), new TypeReference<HashMap<String, Object>>() { });
-      return packageJson;
+
+      try {
+        return objectMapper.readValue(bos.toByteArray(), new TypeReference<HashMap<String, Object>>() { });
+      }
+      catch (JsonParseException e) {
+        // fallback
+        if (e.getMessage().contains("Invalid UTF-8 middle byte")) {
+          log.debug("JSON UTF-8 problem for package {} : {}, attempting with charset ISO-8859-2...",
+              file.getRepositoryItemUid(), e.getMessage());
+          // try again, but assume ISO8859-1 encoding now, that is illegal for JSON
+          return objectMapper.readValue(new String(bos.toByteArray(), Charsets.ISO_8859_1),
```

Comment:
No, the problem was that tarball contained package.json contains invalid UTF 8 character. It actually contains Latin1/ISO-8859-1 encoded character "é". It's exactly this case:
http://jackson-users.ning.com/forum/topics/utf-8-convertion-issues